utils/torch_solver.py

```diff
@@ -9,18 +10,14 @@
     import joblib
     import torch
     from torchvision import transforms
-    from pytorch_lightning import Trainer
 
-    BenchoptCallback = import_ctx.import_from(
-        'torch_helper', 'BenchoptCallback'
-    )
     AugmentedDataset = import_ctx.import_from(
-        'torch_helper', 'AugmentedDataset'
+        'lightning_helper', 'AugmentedDataset'
     )
 
 
 class TorchSolver(BaseSolver):
-    """Torch base solver"""
+    """Pytorch based solver"""
```

Comment:
```suggestion
    """Pytorch base solver"""
```

For consistency: basically I am saying that is a "base solver"